CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ We have several packages which live in this repository. Changes are tracked sepa
 > A highly efficient logging framework that targets resource-constrained devices, like microcontrollers
 
 [defmt-next]: https://github.com/knurling-rs/defmt/compare/defmt-v0.3.9...main
+[defmt-next]: https://github.com/knurling-rs/defmt/pull/894
```

Comment:
We already have a markdown link for `[defmt-next]` which was correct. You can't add another.

The change you've made needs to be listed under `## [defmt-next]` heading.